Makefile

```
@@ -64,6 +65,16 @@ docker: ## builds docker images with the current version and latest tag
 docker-latest: ## builds docker images with latest tag
 	docker buildx build --push --platform linux/amd64,linux/arm64,linux/386 -t yorkieteam/yorkie:latest .
 
+docker-swagger:
```

To display help when running `$ make`

```suggestion
docker-swagger: ## runs swagger-ui with the yorkie api docs
```